### PR TITLE
Use cached cargo-install

### PR DESCRIPTION
**Pull request by TriplEight**
I assumed @tomaka  didn't like [`cargo-nextest`](https://nexte.st/index.html) so not suggesting it here.
Here you go, 1sec instead of 2 min
image

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -69,7 +69,10 @@ jobs:
     - uses: actions/setup-node@v2.5.1
       with:
         node-version: '14'
-    - run: cargo install twiggy  # TODO: somehow cache this in order to not compile twiggy every single time
+    - uses: baptiste0928/cargo-install@v1
```
Comment (tomaka) [inline .github/workflows/ci.yml:72]:
```suggestion
    - uses: baptiste0928/cargo-install@v1  # This action ensures that the twiggy compilation is cached.
```

**Review by tomaka — APPROVED** [review]
Thanks

**Comment by tomaka** [comment]
> I assumed @tomaka didn't like [cargo-nextest](https://nexte.st/index.html) so not suggesting it here.

That's not true. I'm cautious against being overly reliant on tools like this, not avoiding using them. For example parsing the programmatic output of `cargo nextest` wouldn't be a good idea.

**Comment by TriplEight** [comment]
just nagging you there :)

> For example parsing

[yes](https://github.com/nextest-rs/nextest/issues/41), [indeed](https://github.com/nextest-rs/nextest/issues/20), but do you parse it?

**Comment by tomaka** [comment]
What I mean is that if `cargo nextest` happened to be abandoned over night, we should be able to switch back to `cargo test` easily. If we were analyzing the output of `cargo nextest` and performing important actions depending on the result, we would lose the possibility to easily switch back to `cargo test`.

**Comment by TriplEight** [comment]
yeah, this is a normal concern, but considering how rarely we end up parsing i.e. test outputs I think it's worth discussing this probability when we're about to parse some outputs.

**Pull request merged**